lib/multi-acc-new-eks-mixed-observability-pattern/pipeline.ts

```diff
@@ -0,0 +1,388 @@
+                "xray:GetInsightImpactGraph",
+                "ssm:GetParameter"
+            ],
+            resources: ["*"]
+        };
+
+        // creating constructs
+        const ampConstruct = new AmpMonitoringConstruct();
+        const blueprintAmp = ampConstruct.create(scope, context.prodEnv1.account, context.prodEnv1.region);
+        const blueprintCloudWatch = new CloudWatchMonitoringConstruct().create(scope, context.prodEnv2.account, context.prodEnv2.region, PROD2_ENV_ID);
+        const blueprintAmg = new GrafanaOperatorConstruct().create(scope, context.monitoringEnv.account, context.monitoringEnv.region);
+
+        // Argo configuration per environment
+        // CHANGE ME FINALLY
+        // const prodArgoAddonConfig = createArgoAddonConfig(
+        //     'https://github.com/aws-observability/aws-observability-accelerator.git',
+        //     'artifacts/sample-apps/envs/prod',
+        //     'main',
+        //     'public'
+        // );
+        const prodArgoAddonConfig = createArgoAddonConfig(
+            'https://github.com/iamprakkie/aws-observability-accelerator.git',
+            'artifacts/sample-apps/envs/prod',
+            'artifacts',
+            'public'
+        );
+
+        // CHANGE ME FINALLY HERE AS WELL AS IN APP'S VALUES.YAML
+        // const grafanaOperatorArgoAddonConfig = createGOArgoAddonConfig(
+        //     'https://github.com/aws-observability/aws-observability-accelerator.git',
+        //     'artifacts/sample-apps/grafana-operator-app',
+        //     'main',
+        //     'private'
+        // );
+        const grafanaOperatorArgoAddonConfig = createGOArgoAddonConfig(
+            'https://github.com/iamprakkie/aws-observability-accelerator.git',
+            'artifacts/sample-apps/grafana-operator-app',
+            'artifacts',
+            'private'
+        );
+
+        const AmgIamSetupStackProps: AmgIamSetupStackProps = {
+            roleArn: amgWorkspaceIAMRoleARN,
+            accounts: [context.prodEnv1.account!, context.prodEnv2.account!]
+        };
+
+        // get CodePipeline Source Github info
+        const pipelineSrcInfo = JSON.parse(await getSSMSecureString('/cdk-accelerator/pipeline-git-info',this.pipelineRegion))['pipelineSource'];
+        const gitOwner = pipelineSrcInfo.gitOwner;
+        const gitRepositoryName = pipelineSrcInfo.gitRepoName;
+        const gitBranch = pipelineSrcInfo.gitBranch;
+
+        const pipeline = blueprints.CodePipelineStack.builder()
+            .application("npx ts-node bin/multi-acc-new-eks-mixed-observability.ts")
+            .name("multi-account-COA-pipeline")
+            .owner(gitOwner)
+            .codeBuildPolicies([ 
+                new iam.PolicyStatement({
+                    resources: ["*"],
+                    actions: [    
+                        "sts:AssumeRole",
+                        "secretsmanager:GetSecretValue",
+                        "secretsmanager:DescribeSecret",
```

Comment:
Please move this as a file ourside and render it. Check bluperints addons for examples

Reply:
done - moved outside